.github/workflows/release.yml

```diff
@@ -1,11 +1,16 @@
-name: "Release"
+name: 'Release'
 on:
   release:
     types:
       - prereleased
+concurrency:
```

Comment:
This is a bad idea. We will end up with partial releases if we do this, which isn't desirable.

Reply:
How about limiting to one release job ata  time? So we don't get the build servers croweded out by this job?

Reply:
I've removed the `cancel-in-progress`, so that we are limited to one release build going at a time, but they all will get scheduled.

Reply:
Sounds good.

Reply:
We could schedule a run per day? To capture everything within a 24 hour period. Or some cadence that seems reasonable.

Reply:
To stick with the concurrency route, the ID used could be something along the lines of 
```
  group: ${{ github.workflow }}-<releasenumber> 
```
This would not however limit multiple releases running all at once.